app/views/posts/index.html.erb

```diff
@@ -1 +1,23 @@
-<h1>Here is the list of all posts of a user</h1>
```

Comment:
Please add `rails db: create, rails db:migrate, and rails db:seed` and `yarn install` commands in the installation instructions of README.